ipam/calico-ipam.go

```diff
@@ -69,12 +69,26 @@ type ipamArgs struct {
 	IP net.IP `json:"ip,omitempty"`
 }
 
+func updateNodename(conf utils.NetConf) string {
```

Comment:
naming nit: this isn't _updating_ nodename, it's determining it.

Maybe `determineNodename` would be better.